infra/testing/test-server.js

```diff
@@ -47,6 +47,13 @@ app.get('/test/uniqueETag', (req, res) => {
   res.send('ignored');
 });
 
+let uniqueValue = 0;
+app.get('/test/uniqueValue', (req, res) => {
+  res.header('Cache-Control', 'no-cache');
+  res.header('ETag', uniqueValue++);
```

Comment:
FWIW, there shouldn't be a need to add in an `ETag` header here.

(I needed it for the other test case helper route because `broadcast-cache-update` works by comparing ETag header values.)